src/coordinates_photo.py

```diff
@@ -155,22 +157,138 @@ def convert_coordinates(observer_location, observer_time, object_coordinates):
+    
+    r, theta = altaz_to_polar(alt_az.alt.rad, alt_az.az.rad) 
+    print(f"r: {r} m\ntheta: {theta} rad /{math.degrees(theta)} deg \n")
+
+    # en m
+    x,y = polar_to_cartesian(r,theta)
+    print(f"x: {x} m, y:{y} m")
+
+    # en rad
+    object_lat, object_lon = cartesian_to_lon_lat(x,y)
+    object_lat, object_lon = math.degrees(object_lat) , math.degrees(object_lon)
+    # print(f"lon: {lon} rad / {math.degrees(lon)}deg, lat: {lat} rad/{math.degrees(lat)}deg")
+
+    # object_lat, object_lon = method1(alt_az.alt.rad,alt_az.az.rad)
 
-    object_lat = observer_lat
-    object_lon = observer_lon
     return object_lat, object_lon
 
-def main():
+def altaz_to_polar(alt, az):
+    """
+    Submethod to convert altitude (radians), azimuth (radians) of an object to polar coordinates (r,theta)
+    Returns
+    -------
+    (r, theta) : tuple
+    """
+    alt_objet = 10000 # hypothesis: flying at 10km altitude
+    r = math.cos(alt) * alt_objet # en mètres
+    theta = az
+    return r, theta
 
-    # observer_location = (35.6895, 51.3896, 0)
-    # observer_time = "2021-06-21 00:00:00"
-    # object_coordinates = (0, 0)
-    # object_location = convert_coordinates(observer_location, observer_time, object_coordinates)
-    # print(f"The GPS coordinates of the object are: {object_location}")
+def polar_to_cartesian(r, theta):
+    """
+    Submethod to convert radius (radians), theta (radians) of an object 
+    to cartesian coordinates (x,y,z) in meters
+    Returns
+    -------
+    (x, y, z) : tuple
+    """
+    x = r * math.sin(theta) # axis: 
+    y = r * math.cos(theta)
+
+    return x,y
+
+def cartesian_to_lon_lat(x,y):
+    """
+    Submethod to convert cartesian coordinates (x,y,z) (we assume z = 10km) 
+    to longitude, latitude in radians
+    Returns
+    -------
+    (lon, lat) : tuple
+    """
+    z = 10000 # en m
+    lon = math.atan(y/x)
+    lat = math.atan(z/(math.sqrt(x*x+y*y)))
+    return lon, lat
+
+def get_celestial_coordinates(x,y,path):
+    """
+    Submethod to compute celestial coordinates of a given pixel x,y in the image 
+    Returns 
+    -------
+    (Ra,Dec) : tuple
+    """
+
+    ref_pixel,ref_celestial,matrix = get_calibrated_camera_parameters(path)
+
+    delta_Ra = matrix[0][0] * (x - ref_pixel[0]) + matrix[0][1] * (y - ref_pixel[1])
+    delta_Dec = matrix[1][0] * (x - ref_pixel[0]) + matrix[1][1] * (y - ref_pixel[1])
+
+    Ra = ref_celestial[0] + delta_Ra
+    Dec = ref_celestial[1] + delta_Dec
+
+    return Ra,Dec
+
+
+def main():
+    
+    # Prompting the user for observer location
+    observer_location_input = input("Enter observer location (latitude, longitude, altitude): ")
+    # Splitting the input string into latitude, longitude, and altitude
+    observer_location = tuple(map(float, observer_location_input.split(',')))
+
+    # Prompting the user for observer time
+    observer_time = input("Enter observer time (YYYY-MM-DD HH:MM:SS): ")
```

Comment:
Check that the input is in the right format, otherwise weird behavior might happen.